lib/java_buildpack/framework/dyna_trace_agent.rb

```diff
@@ -41,7 +41,7 @@ def release
 
       # (see JavaBuildpack::Component::VersionedDependencyComponent#supports?)
       def supports?
-        @application.services.one_service? FILTER, 'server'
+        (@application.services.one_service? FILTER, 'server') && !(@application.services.one_service? FILTER, 'tenant')
```

Comment:
This filter should mirror the positive filter from the Ruxit framework; it needs to require both `tenant` _and_ `tenanttoken`.